### ISSUE TITLE
Show Alerts Window

### ISSUE THREAD

**Issue by DL2KI** [foldynl/QLog#147]
Which criterias are used to display the recorded allerts in the window? 

Example:

This alert is displayed in the footer of the main window. Time approx. 16:50 UTC.

![Screenshot_20221230_175013](https://user-images.githubusercontent.com/26311402/210094671-30dd0bfb-b376-4f7d-8c16-f2663cc211bc.png)

When the "Show Alerts" window is opened, the entry appears at the second position in the list.

![Screenshot_20221230_174954](https://user-images.githubusercontent.com/26311402/210094686-c55efdb2-bbf5-4484-a32b-f70e6e6f28b7.png)

Three alert rules are active.

**Comment by foldynl** [comment]
The logic is that if the callsign does not exist, it is added to the first position. If it already exists, only the "Updated" count is updated (its position is not changed)

**Comment by DL2KI** [comment]
Yes, thank you for the information. 

You can do it that way. No idea what logics there are and whether one is better than the other, hi. It is important to know how it works.

**Comment by foldynl** [comment]
the logic is that I tried several approaches to display the data. I also tried the approach that dates are displayed according to time. Unfortunately, all variants led to the fact that they were unclear.

It is necessary to know that data can change very quickly - especially in the case of RBN. In these cases, it is necessary to keep order -the  reordering the records after each update  leads to chaos. Therefore, I chose the approach.
